web/js/hide_table_rows.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
+    var $chk = $('#grpChkBox input:checkbox');
```

Comment:
Any reason I’m missing to use $ for variable names? If it’s a convention we have, I totally missed it :)